src/language-handlebars/utils.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+function isGlimmerComponent(node) {
+  if (node.type !== "ElementNode") {
+    return false;
+  }
+
+  const tagFirstChar = node.tag && node.tag[0];
+  const isLocal = node.tag.indexOf(".") !== -1;
```

Comment:
This will be a eslint error, when merge `master` into `next`,  how about commit to `next`

Reply:
Out of curiosity: why https://github.com/prettier/prettier/pull/6967 did not target master?

Reply:
https://github.com/prettier/prettier/pull/6968#issuecomment-554629174